README.md

```diff
@@ -127,11 +127,53 @@ Ideas for future work:
 ## Contributing
 
 The goal of this project is to expand Dart GraphQL ecosystem
-and building a community around this vendor-neutral implementation.
-
-That said, this project is in it's early days and the best
-way to contribute currently is to open issues with questions
-about current and future scope and features of this project.
+and to build a community around this vendor-neutral implementation.
+
+Community contributions are welcome.
+
+### `multipack`
+This repo comes with an unpublished tool called `multipack`. To activate it run the following command.
+```bash
+pub global activate --source path ./multipack
+```
+
+`multipack` provides a simple way of running commands in multiple packages at once. It builds a directed graph of packages
+to run commands in topological order.
+```text
+Available commands:
+  analyze   Run analyzer.
+  exec      Execute any command.
+  fmt       Run formatter.
+  pub       Run pub.
+  pubspec   Update pubspec.yaml
+``` 
+
```

Comment:
````suggestion

`pubspec` has 3 subcommands:
```
Available subcommands:
  clean           cleans dependency overrides
  hard_override   overrides dependencies for local packages
  override        overrides dependencies for local packages
```
````